content/english/Kusto-KC7/activity-1.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Activity 1: Login & Setup of the Cyber Environment"
+draft: false
+weight: 4
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/VL7fJr54CG0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+## Getting Started
+
+Okay Cyber Defenders! The first thing you need to do is set up the environment in which you will be hunting hackers. In order to do that, open the links below and follow the directions on first setting up Azure Data Explorer (ADX) and then logging into the Scoreboard session.
+
+
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/Resources.png?raw=true" alt= “Resources” width="60%" height="value">
+
+### Setting Up Azure Data Explorer (ADX)
+
+ADX is the primary tool used by some cyber defenders for data exploration and analysis. The great thing about ADX is that it is used by cyber analysts at many of the smallest and largest organizations in the world. 
+
+Let’s get you logged in and started with ADX:
+
+1.	Go to [Azure Data Explorer] https://dataexplorer.azure.com/ and login with your Microsoft account 
```

Comment:
```suggestion
1.	Go to [Azure Data Explorer](https://dataexplorer.azure.com/) and login with your Microsoft account 
```

Reply:
This works ok for me, but doesn't for Scoreboard so for consistency I'm removing the ()